consensus/safety-rules/src/counters.rs

```diff
@@ -45,3 +47,8 @@ pub fn start_timer(source: &str, field: &str) -> HistogramTimer {
 pub fn set_state(field: &str, value: i64) {
     STATE_GAUGE.with_label_values(&[field]).set(value);
 }
+
+#[allow(dead_code)]
+pub fn get_state(field: &str) -> i64 {
+    STATE_GAUGE.with_label_values(&[field]).get()
+}
```

Comment:
If it's dead code, you can just remove it.

Reply:
Replace `#[allow(dead_code)]` with `#[cfg(any(test))]`.